tests/unit/package_managers/yarn/test_main.py

```diff
@@ -313,7 +318,28 @@ def test_set_yarnrc_configuration(
     mock_write.assert_called_once()
 
 
-def test_verify_yarnrc_paths() -> None:
+@mock.patch("cachi2.core.package_managers.yarn.project.YarnRc.write")
+def test_disable_constraints_checks_in_yarn_v4(
+    mock_write: mock.Mock,
+) -> None:
+    yarn_rc = YarnRc(mock.Mock(), {})
+
+    package_json = mock.Mock()
+    package_json.package_manager = "yarn@4.0.0"
+
+    project = mock.Mock()
+    project.yarn_rc = yarn_rc
+    project.package_json = package_json
+    output_dir = RootedPath("/tmp/output")
```

Comment:
nitpick: this could be replaced by `rooted_path` fixture

Reply:
Fixed